tests/ui/typeck/minus-string.rs

```diff
@@ -0,0 +1,7 @@
+// Regression test for issue #813 (commit de4b383).
```

Comment:
Nit: drop the commit, just keep the issue, the commit isn't interesting for the purposes of this test.